### PR TITLE
FISH-6522

### PR DESCRIPTION
**Pull request by luiseufrasio**
[CVE-2021-31684] added in Security Fixes Summaries

### CODE REVIEW

File: community/docs/modules/ROOT/pages/Security/Security Fix List.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@ The following is a list of tracked _**C**ommon **V**ulnerabilities and **E**xpos
 |=======================================================================
 |ID |CVSS v3.0 Base Score |Status |Summary |Release |Pull Requests |Observations
 
+|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31684[CVE-2021-31684] | N/A | FIXED | A vulnerability was discovered in the indexOf function of JSONParserByteArray in JSON Smart versions 1.3 and 2.4 which causes a denial of service (DOS) via a crafted web request. | 5.2022.3 | https://github.com/payara/Payara/pull/5944[#5944] | Fixed by upgrading nimbus-jose-jwt to 9.25
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Security/Security Fix List.adoc:10]:
```suggestion
|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31684[CVE-2021-31684] | N/A | FIXED | A vulnerability was discovered in the indexOf function of JSONParserByteArray in JSON Smart versions 1.3 and 2.4 which causes a denial of service (DOS) via a crafted web request. | 5.2022.4 | https://github.com/payara/Payara/pull/5944[#5944] | Fixed by upgrading nimbus-jose-jwt to 9.25
```

File: enterprise/docs/modules/ROOT/pages/Security/Security Fix List.adoc [hunk 2]
```diff
@@ -7,6 +7,8 @@ The following is a list of tracked _**C**ommon **V**ulnerabilities and **E**xpos
 |=======================================================================
 |ID |CVSS v3.0 Base Score |Status |Summary |Release |Observations
 
+|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31684[CVE-2021-31684] | N/A | FIXED | A vulnerability was discovered in the indexOf function of JSONParserByteArray in JSON Smart versions 1.3 and 2.4 which causes a denial of service (DOS) via a crafted web request. | 5.43.0 | Fixed by upgrading nimbus-jose-jwt to 9.25
```
Comment (JamesHillyard) [inline enterprise/docs/modules/ROOT/pages/Security/Security Fix List.adoc:10]:
```suggestion
|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31684[CVE-2021-31684] | N/A | FIXED | A vulnerability was discovered in the indexOf function of JSONParserByteArray in JSON Smart versions 1.3 and 2.4 which causes a denial of service (DOS) via a crafted web request. | 5.44.0 | Fixed by upgrading nimbus-jose-jwt to 9.25
```

**Review by JamesHillyard — CHANGES_REQUESTED** [review]

File: enterprise/docs/modules/ROOT/pages/Security/Security Fix List.adoc (hunk 2 above)
Comment (luiseufrasio) [reply enterprise/docs/modules/ROOT/pages/Security/Security Fix List.adoc]: done!

File: community/docs/modules/ROOT/pages/Security/Security Fix List.adoc (hunk 1 above)
Comment (luiseufrasio) [reply community/docs/modules/ROOT/pages/Security/Security Fix List.adoc]: done

**Pull request merged**